src/Skoruba.IdentityServer4.STS.Identity/appsettings.json

```diff
@@ -9,26 +9,19 @@
     "ProviderType": "SqlServer"
   },
   "CertificateConfiguration": {
-
```

Comment:
Hmm, automatic formatting. I can set this back to the way it was if it's better that way.